tensorflow/core/kernels/depthtospace_op.cc

```diff
@@ -143,6 +127,92 @@ class DepthToSpaceOp : public OpKernel {
   TensorFormat data_format_;
 };
 
+// Template specialization for GPUDevice, explicit referncing GPUDevice in code
```

Comment:
End comment with period, also typo in `referncing`

Reply:
Done